### PR TITLE
Be able to configure memory limits trough config

### PR DESCRIPTION
**Pull request by daniquilez**

### CODE REVIEW

File: templates/odoo.conf.j2 [hunk 1]
```diff
@@ -56,6 +56,10 @@ workers = {{ odoo_role_workers }}
 limit_time_cpu = {{ odoo_role_limit_time_cpu }}
 limit_time_real = {{ odoo_role_limit_time_real }}
 
+; Customize the Odoo memory limits
+limit_memory_hard : {{ odoo_role_limit_memory_hard }}
```
Comment (danypr92) [inline templates/odoo.conf.j2:60]: ``
```suggestion
limit_memory_hard = {{ odoo_role_limit_memory_hard }}
```

File: templates/odoo.conf.j2 [hunk 2]
```diff
@@ -56,6 +56,10 @@ workers = {{ odoo_role_workers }}
...
+limit_memory_soft : {{ odoo_role_limit_memory_soft }}
```
Comment (danypr92) [inline templates/odoo.conf.j2:61]:
```suggestion
limit_memory_soft = {{ odoo_role_limit_memory_soft }}
```

File: defaults/main.yml [hunk 3]
```diff
@@ -81,6 +81,10 @@ odoo_role_enable_dbfilter_from_header: false
 odoo_role_limit_time_cpu: 60
 odoo_role_limit_time_real: 120
 
+# Customize the Odoo memory limits
+odoo_role_limit_memory_hard : 2684354560
```
Comment (danypr92) [inline defaults/main.yml:85]: The default value in Odoo is: 2048 * 1024 * 1024 = 2147483648

https://git.coopdevs.org/coopdevs/odoo/OCB/-/blob/12.0/odoo/tools/config.py#L303

```suggestion
odoo_role_limit_memory_hard : 2147483648
```

File: defaults/main.yml [hunk 4]
```diff
@@ -81,6 +81,10 @@ odoo_role_enable_dbfilter_from_header: false
...
+odoo_role_limit_memory_soft : 2147483648
```
Comment (danypr92) [inline defaults/main.yml:86]: The default value in Odoo is: 2048 * 1024 * 1024 = 2147483648

https://git.coopdevs.org/coopdevs/odoo/OCB/-/blob/12.0/odoo/tools/config.py#L307

```suggestion
odoo_role_limit_memory_soft : 2147483648
```

**Review by danypr92 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -96,6 +96,18 @@ odoo_role_odoo_proxy_mode: false
...
+* Odoo other server settings
+
+```yml
+# Customize the Odoo timeouts
+odoo_role_limit_time_cpu: 60
+odoo_role_limit_time_real: 120
+
+# Customize the Odoo memory limits
+odoo_role_limit_memory_hard : 2684354560
+odoo_role_limit_memory_soft : 2147483648
```
Comment (oyale) [inline README.md:108]:
```suggestion
odoo_role_limit_memory_hard: 2684354560
odoo_role_limit_memory_soft: 2147483648
```

**Review by oyale — APPROVED** [review]

**Pull request merged**